recipes/nlopt/bld.bat

```diff
@@ -0,0 +1,74 @@
+@echo off
+
+@rem See https://github.com/jyypma/nloptr/blob/master/INSTALL.windows
+cp "%RECIPE_DIR%\CMakeLists.txt" .
+if errorlevel 1 exit 1
+cp "%RECIPE_DIR%\config.cmake.h.in" .
+if errorlevel 1 exit 1
+
+mkdir build
+cd build
+
+set CMAKE_CONFIG="Release"
+
+:: Set the right msvc version according to Python version
+REM write a temporary batch file to map cl.exe version to visual studio version
+echo @echo 15=9 > msvc_versions.bat
+echo @echo 16=10 >> msvc_versions.bat
+echo @echo 17=11 >> msvc_versions.bat
+echo @echo 18=12 >> msvc_versions.bat
+echo @echo 19=14 >> msvc_versions.bat
+
+REM Run cl.exe to find which version our compiler is
+for /f "delims=" %%A in ('cl /? 2^>^&1 ^| findstr /C:"Version"') do set "CL_TEXT=%%A"
+FOR /F "tokens=1,2 delims==" %%i IN ('msvc_versions.bat') DO echo %CL_TEXT% | findstr /C:"Version %%i" > nul && set VSTRING=%%j && goto FOUND
+EXIT 1
+:FOUND
+
+call :TRIM VSTRING %VSTRING%
+
+set NLOPT_ARCH=x86
+if "%VSTRING%" == "9" (
+    set GENERATOR=Visual Studio 9 2008
+    set NLOPT_VC=vc9
+)
+if "%VSTRING%" == "10" (
+    set GENERATOR=Visual Studio 10 2010
+    set NLOPT_VC=vc10
+)
+if "%VSTRING%" == "14" (
+    set GENERATOR=Visual Studio 14 2015
+    set NLOPT_VC=vc14
+)
+if not defined GENERATOR EXIT 1
+
+if %ARCH% EQU 64 (
+    set GENERATOR=%GENERATOR% Win64
+    set NLOPT_ARCH=x64
+)
+
+cmake .. -LAH -G"%GENERATOR%"                       ^
+    -DPYTHON_EXECUTABLE="%PYTHON%"                  ^
+    -DPYTHON_INCLUDE_PATH="%PREFIX%\include"        ^
+    -DPYTHON_LIBRARY="%PREFIX%\libs\python27.lib"   ^
```

Comment:
Windows bindings can't be built?  Then why have the python stuff at all?  Do the bindings work only on 2.7?  If that's the case, I'd suggest limiting this recipe to only build for 2.7 right now on Win.